internal/telemetry/cluster.go

```diff
@@ -202,6 +203,19 @@ func (c *Collector) IsPlusEnabled() bool {
 	return c.Config.IsPlus
 }
 
+// InstallationFlags returns the list of all flags
+func (c *Collector) InstallationFlags() []string {
+	flags := os.Args[1:]
+	// tests that compare the report currently break with this addition,
+	// this is temp fix until tests are refactored
+	for _, flag := range flags {
+		if flag == "-test.paniconexit0" {
```

Comment:
May be worth to check: `https://pkg.go.dev/slices#Contains` It already implements the loop